src/legacy/core_plugins/metrics/public/components/aggs/percentile_rank/percentile_rank.js

```diff
@@ -89,21 +99,14 @@ export const PercentileRankAgg = props => {
             />
           </EuiFormRow>
         </EuiFlexItem>
-        <EuiFlexItem>
-          <EuiFormRow
-            id={htmlId('value')}
-            label={(<FormattedMessage
-              id="tsvb.percentileRank.valueLabel"
-              defaultMessage="Value"
-            />)}
-          >
-            <EuiFieldText
-              value={model.value}
-              onChange={handleTextChange('value')}
-            />
-          </EuiFormRow>
-        </EuiFlexItem>
       </EuiFlexGroup>
+      <EuiSpacer/>
+      <PercentileRankValues
+        disableAdd={isTablePanel}
+        disableDelete={isTablePanel}
+        model={isTablePanel ? [last(percentileRankValueRows)] : percentileRankValueRows}
```

Comment:
If you switch to the table panel and change the value of the percentile rank, it will wipe the other values you have configured (e.g. in timeseries panel) - is this intended? IMHO it should just change the last value.